### ISSUE TITLE
Login form flicker and unexpected password manager auto-fill behaviour

### ISSUE THREAD

**Issue by booni3** [livewire/livewire#943]
As you can see in the below video:
- when we refresh the page, the password is auto-filled by my password manager (last pass in this case). 2 requests are sent to the server and a flicker occurs.

![Kapture 2020-05-09 at 10 18 40](https://user-images.githubusercontent.com/20134485/81469784-e3de8600-91de-11ea-8d7e-7423e00f166b.gif)

This is in desktop, chrome. The same happens in Safari too.

Also on iPhone we get other unexpected behaviour when trying to autofill the password. The email and password fields fill but then immediately disappears again, making the password manager useless.

I use the autofill for almost everything and have never experienced this happening before, so expect it must be the way in which livewire sends and updates the PHP variables. My guess is that it is something to do with auto-filling 2 fields at exactly the same time.

Things I have tried:

- `wire:ignore` on inputs or a parent div around the inputs. Data binding on wire:model is still present which I believe is causing the issue.

- `wire.model.debounce.Xms` in varying amounts of debouncing and offsetting each field. This does actually help on desktop. It prevents the flicker but does result in only 1 field being filled by the password manager. On iPhone Safari it does not help at all (desktop/chrome video below showing behaviour)

![Kapture 2020-05-09 at 11 03 27](https://user-images.githubusercontent.com/20134485/81470807-bd234e00-91e4-11ea-89a0-d60be2e1155c.gif)

- `wire:model.lazy` makes no difference

Other ideas:

- Remove `wire:model` data binding and only send data on form submit. I am not sure if this can be done with Livewire natively or if we would need to use Alpine?

**Comment by booni3** [comment]
I have tested my 3rd option to remove wire:model binding and just submit the data with the form. This seems to work ok but its messy... is there a nicer way?

```blade
<form x-data="{email: '', password: ''}">
  <input name="email" x-model="email">
  <input name="password" x-model="password">
  <button @click.prevent="@this.set('email', email);@this.set('password', password);@this.call('login')">Login</button>
</form>
```

I realise we can include the variables inside the @this.call, but I want to make use of livewires `$this->validate` call and therefore we need to have the variables set inside the model first.

**Comment by lancepioch** [comment]
This should be resolved in v1.1.0. Let us know if it's not, thanks!

**Comment by Neeraj1005** [comment]
The same issue occur in v2.6. Is anyone has any idea how to resolve this...?
